src/seedu/addressbook/data/person/Person.java

```diff
@@ -87,4 +87,14 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables) {
```

Comment:
Great that you've implemented the `getPrintableString(Printable... printables)` to take in multiple printable to be printed, however, it seems that you did not use the method anywhere. Do find out where can you use this method at if not, there's no point in doing so.